console/src/test/java/org/web3j/console/project/ProjectStructureTest.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2019 Web3 Labs LTD.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package org.web3j.console.project;
+
+import java.io.File;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import org.web3j.TempFileProvider;
+
+public class ProjectStructureTest extends TempFileProvider {
+    private ProjectStructure projectStructure;
+
+    private void init() throws Exception {
+        setUp();
```

Comment:
you do not need to call `setUp` because it is annotated with `@Before`

but you need this to run before `init()`

You should override the base `setUp()` and call `super.setUp()` in the first line to have access to the temp file

Reply:
done